bindings_ffi/src/mls.rs

```diff
@@ -4129,7 +4129,7 @@ mod tests {
         message_callbacks.wait_for_delivery(None).await.unwrap();
         assert_eq!(bo.provider.db().intents_published(), 4);
 
-        assert_eq!(message_callbacks.message_count(), 5);
+        assert_eq!(message_callbacks.message_count(), 6);
```

Comment:
Making metadata non-async increased this to 6.. strangely enough. It appears that with the thread no longer yielding, the thread retains control until the 6th message is processed.

Reply:
returning 6 on main as well, sans async metadata

https://github.com/xmtp/libxmtp/blob/main/bindings_ffi/src/mls.rs#L4151